examples/query-for-document/src/opa/policies/rules/type_rule.rego

```diff
@@ -0,0 +1,12 @@
+package type_rule
+
+
+presents_packaging = {
+    "allow": true,
+    "reason": sprintf("%v may pack presents for children", [input.name])
+} {
+    input.santaPartyMember == "SANTA"
```

Comment:
So, only Santa may pack presents then? :)

Reply:
I will use array of Santa and Elf